### DISCUSSION TITLE
Color mapping?

### DISCUSSION THREAD

**Discussion by abadire** [mantinedev/mantine#1752] [Q&A]
Hi!

How could I map colors from default theme? For example I want to make button look like an `Input`, give it the same color, border and so on. I can hardcode color values taken from dev-tools, but it won't work if I change app's colors later. So right now I pick the default color, search Mantine sources for it, count it's position in the color array and use `theme.colors.<color>[<number>]`. Is there a better solution? Or am I missing something?

**Comment by rtivital** [comment]
Use Input with custom component – https://mantine.dev/core/input/#custom-component
